src/components/Trackpad/ControlBar.tsx

```
@@ -126,11 +130,17 @@ export const ControlBar: React.FC<ControlBarProps> = ({
                 <MouseIcon side="R" />
             </button>
 
-            <button className={baseBtn}>
+            <button
+                className={baseBtn}
+                onPointerDown={(e) => prevent(e, onCopy)}
+            >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)